.gitignore

```diff
@@ -30,3 +30,4 @@
 *.cmake
 /Testing
 /install_manifest*.txt
+Makefile
```

Comment:
How about: `*Makefile`?

Reply:
This project generates only one `Makefile`, so I'd rather be explicit.

Reply:
Yes, but what if you build out of source :)

Reply:
Right. But `Makefile` (without the `/` prefix) will apply to any directory, right? It won't apply to `somethingMakefile` but that is okey.

Reply:
Oh, haha. TIL